app/views/account/tasks/move.js.erb

```diff
@@ -0,0 +1,11 @@
+$('#incomplete_tasks').html('');
+<%- @incomplete_tasks.each do |task| %>
+var task_html = $("<%= j render('/account/projects/show_tasks', task: task, project: @project) %>");
+$('#incomplete_tasks').append(task_html);
+<% end %>
+
+$('#complete_tasks').html('');
```

Comment:
render only required section (if task is in completed - re-render only completed)